### PR TITLE
feat: Upgrade rosetta to v0.7.2

### PR DESCRIPTION
**Pull request by fkneeland-figure**
## Description

Upgrade Rosetta to the latest version so that provenance can past Rosetta tests

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: server/rosetta/client_online.go [hunk 1]
```diff
@@ -481,13 +484,39 @@ func (c *Client) blockTxs(ctx context.Context, height *int64) (crgtypes.BlockTra
 
 func (c *Client) getHeight(ctx context.Context, height *int64) (realHeight *int64, err error) {
 	if height != nil && *height == -1 {
-		genesis, err := c.tmRPC.Genesis(ctx)
+		genesisChunk, err := c.tmRPC.GenesisChunked(ctx, 0)
```
Comment (arnabmitra) [inline server/rosetta/client_online.go:487]: @fkneeland-figure  could we possibly highlight what you changed vs upstream

Comment (channa-figure) [reply server/rosetta/client_online.go:487]: I believe it was only changes to the go.mod file for rosetta and golang-lru versions?

**Review by channa-figure — CHANGES_REQUESTED** [review]

File: server/rosetta/client_online.go (hunk 1 above)
Comment (arnabmitra) [reply server/rosetta/client_online.go:487]: Screen Shot 2021-12-09 at 9 02 13 PM
not sure about the LRU change, but these are the 4 files i see change, seems like an empty string added to server.go, is the only change, i wonder if we should PR against the [`v0.43.`](https://github.com/provenance-io/cosmos-sdk/tree/v0.44.3-pio) branch and then create a reales 0.44-pio.x branch for release? (the diff between `v0.44.3-pio` and this branch is super easy to follow

Comment (arnabmitra) [reply server/rosetta/client_online.go:487]: i guess this https://github.com/provenance-io/cosmos-sdk/compare/provenance-io:release/v0.44.x...provenance-io:v0.44.3-pio plus @fkneeland-figure change of 1 file adds up 🤷

**Review by arnabmitra — APPROVED** [review]

**Review by derekadams — APPROVED** [review]

**Comment by iramiller** [comment]
LGTM

**Comment by fkneeland-figure** [comment]
In order to make our release process for this downstream repo we are going to be using the branch release-pio/v0.44.3.x 

This branch is based off of cosmos main v0.44.3 and will include all changes provenance needs to that version.  I moved this to a separate pr that is based off of that new release branch for our updates to cosmos v0.44.3 and I added links to both of the cosmos prs that have been merged to master that are included in this update: https://github.com/provenance-io/cosmos-sdk/pull/6

**Pull request closed without merging**